src/main/java/org/kuali/kfs/sys/service/impl/GeneralLedgerPendingEntryServiceImpl.java

```diff
@@ -79,8 +79,9 @@
 import java.util.Map;
 
 /*
- * CU Customization: Backported the FINP-7275 fix from the 2020-12-10 financials patch.
- * This overlay can be removed once we upgrade to the 2020-12-10 patch or later.
+ * CU Customization: Backported the FINP-7275 fix from the 2020-12-10 financials patch,
+ * and updated one of the new methods from that release to protected visibility.
+ * This overlay can be removed or simplified once we upgrade to the 2020-12-10 patch or later.
```

Comment:
Should this comment be more clear about the changes needed when we upgrade to 12/10?
I will ask KualiCo to change the method's visibility to protected.

Reply:
I will update the comment for clarity.